redisvl/schema/__init__.py

```diff
@@ -0,0 +1,38 @@
+from redisvl.schema.fields import (
+    BaseField,
+    TagFieldSchema,
+    TextFieldSchema,
+    GeoFieldSchema,
+    NumericFieldSchema,
+    BaseVectorField,
+    HNSWVectorFieldSchema,
+    FlatVectorFieldSchema,
+)
+from redisvl.schema.schema import (
+    StorageType,
+    IndexModel,
+    FieldsModel,
+    Schema,
+    SchemaGenerator,
+    read_schema,
+    SchemaValidationError,
+)
+
+
+__all__ = [
+    "BaseField",
+    "TagFieldSchema",
+    "TextFieldSchema",
+    "GeoFieldSchema",
+    "NumericFieldSchema",
```

Comment:
do these fields need to be exported? I'm assuming only 
- ``Schema``
- ``StorageType``

need to be exported. We want to maintain a wall between what we see and the user sees in the importable space.

Reply:
Yep will redact what I can!